Makefile

```
@@ -583,8 +588,60 @@ clean: stop ## Stop containers and remove volumes
 	@$(MAKE) remove-openrag-images
 	@echo "$(PURPLE)Cleanup complete!$(NC)"
 
-factory-reset: ## Complete reset (stop, remove volumes, clear data, remove images)
-	@echo "$(RED)WARNING: This will completely reset OpenRAG!$(NC)"; \
+factory-reset: ## Complete reset (stop, remove volumes, clear data)
```

checkmake (0.3.2)

[warning] Target body for "factory-reset" exceeds allowed length of 5 lines (50).

(maxbodylength)